andela/hc-silicon
# [Feature 157694753] System nagging and emails

### What does this PR do?
Sends periodic emails to a user if a check has gone down and has not yet been resolved. 
The user is able to set the nag interval time that the emails can keep being sent.

### Description of Task to be completed?
If a user has checks that are down and have not yet been resolved, the system keeps sending emails to the user at the nag interval set by the user, until the check is resolved and is up and running.

### How should this be manually tested?
Set up a check on healthchecks and set the period, grace and nag times. Once the check goes down, an email will be sent to your mailbox. 
If the check is not resolved and the nag interval has elapsed, another email will be sent to remind you of the unresolved check.
This will go on iteratively until the check is resolved.

### What are the relevant pivotal tracker stories?
- [System nagging](https://www.pivotaltracker.com/story/show/157694753)

### Screenshots
<img width="1206" alt="screen shot 2018-06-07 at 23 24 59" src="https://user-images.githubusercontent.com/29580769/41155124-6cc9ac20-6b25-11e8-8673-f9ec3242738a.png">

<img width="1067" alt="screen shot 2018-06-08 at 14 05 05" src="https://user-images.githubusercontent.com/29580769/41155148-7f09343c-6b25-11e8-840f-07f98fc4273b.png">


@Awesome94 @kzyGit @tonykimathi @muhozi @sam-karis
